dist/etc/conf/nucleo-wl55jc.env

```diff
@@ -0,0 +1,84 @@
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN?=
+
+HIL_DUT_CTS_GPIO?=
+HIL_DUT_CTS_PORT?=
+HIL_DUT_CTS_PIN?=
+
+HIL_DUT_RTS_GPIO?=
+HIL_DUT_RTS_PORT?=
+HIL_DUT_RTS_PIN?=
+
+HIL_DUT_RX_GPIO?="PA2/CN9-7/D1"
+HIL_DUT_RX_PORT?=0
+HIL_DUT_RX_PIN?=2
+
+HIL_DUT_TX_GPIO?="PA3/CN9-8/D0"
```

Comment:
Are you sure this uart is not being used?

Reply:
Oops. Seems like the default PIN mapping was different. Updated in fixup